modules/schematics/src/effect/schema.json

```diff
@@ -63,6 +63,12 @@
       "description":
         "Specifies if effect has api success and failure actions wired up",
       "aliases": ["a"]
+    },
+    "effectCreator": {
+      "type": "boolean",
+      "default": false,
+      "description": "Specifies if the effect creation uses 'createEffect'",
+      "aliases": ["ec"]
```

Comment:
What abous using `c` as alias and reuse it for `createAction`?

Reply:
cc @brandonroberts , because I just saw you were using `ac` for the `createAction` schematics

Reply:
> @itayod it would be easier to generate parts of the template in the factory code and pass a variable to the template for output. That's probably as close as we'll get to partial templates.

I agree that's what I just did 😄

Reply:
wouldn't it make conflicts when generating feature?

Reply:
This is indeed true, it would mean using the one of the two would result in both of the creators being used.

Reply:
I think we should simplify here and just use `--creators` to use the new syntax for functions for actions and effects. Its unlikely that you'll use one without the other, so let's shift people in that direction.